src/emulate.c

```diff
@@ -405,19 +405,8 @@ static void block_translate(riscv_t *rv, block_t *block)
         block->pc_end += ir->insn_len;
         block->n_insn++;
         /* stop on branch */
-        if (insn_is_branch(ir->opcode)) {
-            /* recursive jump translation */
-            if (ir->opcode == rv_insn_jal
-#if RV32_HAS(EXT_C)
-                || ir->opcode == rv_insn_cj || ir->opcode == rv_insn_cjal
-#endif
-            ) {
-                block->pc_end = block->pc_end - ir->insn_len + ir->imm;
-                ir->branch_taken = ir + 1;
-                continue;
-            }
+        if (insn_is_branch(ir->opcode))
```

Comment:
Shall we change `master` branch accordingly?

Reply:
Yes, this change is also required for detecting memset and memcpy.